contracts/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
doesnt make it more sense to put all these configs into the `contracts` directory?

Reply:
Yes, make sense

Reply:
All the configs are on the contracts level already.